ddtrace/contrib/pytest_bdd/plugin.py

```diff
@@ -25,6 +25,39 @@ def _store_span(item, span):
     setattr(item, "_datadog_span", span)
 
 
+def _extract_step_func_args(step, step_func, step_func_args):
```

Comment:
I'm thinking of adding a unit test for this, even if it's fairly directly tested by existing tests.

Reply:
It's your call, if it's already covered I think it's enough